test/plugin/scenarios/servicecomb-0.x-scenario/support-version.list

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# lists your version here
+
+0.1.0
+0.2.0
+0.3.0
+0.4.0
+0.5.0
```

Comment:
Should have more releases today, could you recheck?

Reply:
No more versions were found in the [central repo](https://mvnrepository.com/artifact/io.servicecomb)

Reply:
I'll add a new scenario for 1.x

Reply:
I think they move to Apache after that :) I don't know whether the plugin works for that or not. cc @WillemJiang Anything related to SkyWalking changed?

Reply:
https://github.com/apache/servicecomb-java-chassis/releases